ecosystem/sep-0009.md

```diff
@@ -6,8 +6,8 @@ Title: Standard KYC / AML fields
 Author: stellar.org
 Status: Active
 Created: 2018-07-27
-Updated: 2020-04-03
-Version 1.2.0
+Updated: 2020-12-14
+Version 1.2.1
```

Comment:
This adds new fields and needs to be a minor version update rather than a patch version.
```suggestion
Version 1.3.0
```